vmm/src/interrupt.rs

```diff
@@ -228,6 +229,19 @@ impl InterruptSourceGroup for KvmMsiInterruptGroup {
                 kvm_route.u.msi.address_hi = cfg.high_addr;
                 kvm_route.u.msi.data = cfg.data;
 
+                #[cfg(target_arch = "aarch64")]
+                {
+                    // TODO:
+                    // Need to check Cap KVM_CAP_MSI_DEVID to determine whether devid is requred.
```

Comment:
When the kvm-ioctls fix is done I think it would be best to just check for that capability rather than make it architecture specific.

Reply:
OK. Good to know that setting `devid` will not impact X86.